src/context/mod.rs

```diff
@@ -499,7 +499,7 @@ impl Context {
         feature = "min-redis-compatibility-version-7-2",
         feature = "min-valkey-compatibility-version-8-0"
     ),))]
-    pub fn set_acl_category(&self, command_name: *const i8, acl_flags: *const i8) -> raw::Status {
+    pub fn set_acl_category(&self, command_name: *const c_char, acl_flags: *const c_char) -> raw::Status {
```

Comment:
This works for me.

I also saw that u8 works as an alternative

Reply:
Thanks for confirming.